google/cloud/internal/rest_retry_loop.h

```diff
@@ -61,41 +62,35 @@ template <
     typename std::enable_if<google::cloud::internal::is_invocable<
                                 Functor, RestContext&, Request const&>::value,
                             int>::type = 0>
-auto RestRetryLoopImpl(std::unique_ptr<RetryPolicy> retry_policy,
-                       std::unique_ptr<BackoffPolicy> backoff_policy,
+auto RestRetryLoopImpl(RetryPolicy& retry_policy, BackoffPolicy& backoff_policy,
```

Comment:
If we are going to make this change, we should do it in `RetryLoopImpl` too.

Reply:
Done.